src/views/CreateView.vue

```diff
@@ -348,12 +369,51 @@ export default {
           unit: this.selectedUnit,
           rest: ''
         }
-        this.isFormSubmitted = false
+        this.isExerciseSubmitted = false
         this.areExercisesDisplayed = false
         this.selectedExercise = ''
       } else {
-        this.isFormSubmitted = true
+        this.isExerciseSubmitted = true
       }
+    },
+    async submitWorkout() {
+      if (this.addedExercises.length !== 0) {
+        const { data: { user } } = await supabase.auth.getUser()
+        const { data, error } = await supabase
+          .from('Workouts')
+          .insert([
+            {
+              workout: {
+                workout_name: this.workoutName,
```

Comment:
Just name in here makes more sense

Reply:
The thing is `workout_name` is the name of an appropriate column in the supabase table. So, I'd have to change it there and then change it in all files. And I'm still concerned about `name` as a variable name being unsuitable name for a variable.

So, I'll remember next time to give more appropriate names in the databases like supabase or firebase but for now I'd just leave it as is.

Reply:
**Corrected**